src/notation/internal/notationactioncontroller.h

```diff
@@ -78,6 +78,8 @@ class NotationActionController : public muse::actions::Actionable, public muse::
     INotationUndoStackPtr currentNotationUndoStack() const;
     INotationMidiInputPtr currentNotationMidiInput() const;
 
+    mu::engraving::Score* score() const;
```

Comment:
let's rename to currentNotationScore
in general, such a method should not be in this class :) ideally, these manipulations should soon be moved somewhere else, but for now, let's just rename the method